static/app/views/replays/detail/console/index.tsx

```diff
@@ -36,6 +36,11 @@ function Console({breadcrumbs, startTimestampMs}: Props) {
   const {items, setSearchTerm} = filterProps;
   const clearSearchTerm = () => setSearchTerm('');
   const {currentTime, currentHoverTime} = useReplayContext();
+  // Keep a reference of object paths that are expanded (via <ObjectInspector>)
+  // by log row, so they they can be restored as the Console pane is scrolling.
+  // Due to virtualization, components can be unmounted as the user scrolls, so
+  // state needs to be remembered.
+  const expandPaths = useRef(new Map<number, Set<string>>());
```

Comment:
We might need to reset this array after `filterProps` changes (filter or sort props), because the rowId will point to a different bit of data in that case. Keying off the `items` array should be sufficient.

Reply:
Good catch, we could wrap this in an useEffect, with `items` as a dep?

Reply:
yeah, something like:
```
useEffect(() => {
  expandPaths.current = new Map();
}, [items]);
```

that might be it. i think that would work.